src/plonk/circuit.rs

```diff
@@ -1556,7 +1563,8 @@ impl<'a, F: Field> VirtualCells<'a, F> {
     }
 
     /// Query an Any column at a relative position
-    pub fn query_any(&mut self, column: Column<Any>, at: Rotation) -> Expression<F> {
+    pub fn query_any<C: Into<Column<Any>>>(&mut self, column: C, at: Rotation) -> Expression<F> {
+        let column = column.into();
```

Comment:
Hmm... Now that I think about this more, I'm less sure this change is useful given the existance of specific methods `query_advice` etc. I'm wondering if instead we should remove the `query_any` API entirely, and require callers to call the more specific column-typed APIs? Are there situations where the chip configurer truly won't know the column types they are querying to use in a gate, and does not need to care about them from the perspective of the returned `Expression`s? cc @ebfull

Reply:
If we _do_ keep `query_any` around, then I agree that we should have it accept `Into<Column<Any>>` to match `enable_equality`.

Reply:
I do agree. Otherways the API looks a bit inconsistent in that regard. 
I left it modified. And will wait for a response in case it needs to be reverted too.

Reply:
We decided to keep the API and take the modification.